release_notes.md

```diff
@@ -0,0 +1,14 @@
+<!-- Please put your changes into the appropriate category (or categories) below. -->
+
+## New Features
+
+## Bug fixes
+- Remove incorrect information from C# docs summary for IDurableEntityClient.ReadEntityStateAsync() regarding states large than 16KB (#1637)
+- Fix a NullReferenceException in IDurableClient.SignalClient() for IDurableClient objects created by the new DurabilityClientFactory (#1644)
```

Comment:
I don't know if these release notes apply to this PR. We should probably leave that out of v1 for now, since v1 release we have less process with anyways.